schemas/test/en/test_currency.json

```diff
@@ -93,10 +93,93 @@
                                         "maximum": {
                                             "value": 1000000
                                         }
+                                    },
+                                    {
+                                        "id": "answer-jpy",
```

Comment:
This has the same id as the Yen input above it, so the value is tied to that input in the store is incorrect, meaning the value entered for this input is discarded and the test for default max value isn't working/enforced. 

https://user-images.githubusercontent.com/108286538/232751997-c6a16124-7a93-440c-baef-cfbe4862a4fc.mov

Reply:
These videos are pretty neat 👍